test/lit/passes/j2cl-merge-itables.wast

```diff
@@ -11,19 +11,20 @@
       (field $vtable (ref $Object.vtable))
       (field $itable (ref $Object.itable)))))
 
+    ;; CHECK:       (type $Object.vtable (sub (struct (field structref))))
```

Comment:
This motion looks like it worsened the readability?

Reply:
I can add --preserve-type-order to this file.